.buildkite/react-native-pipeline.yml

```diff
@@ -17,7 +17,6 @@ steps:
         commands:
           - ./bin/generate-react-native-fixture
         matrix:
-          - "0.71"
```

Comment:
may want to consider rebasing on next as 0.73 has been added to the pipeline now